src/seedsigner/views/view.py

```diff
@@ -188,10 +188,9 @@ def run(self):
             # exiting.
             time.sleep(0.25)
 
-            # Kill the SeedSigner process; systemd will automatically restart it.
-            # `.*` is a wildcard to detect either `python`` or `python3` and with or
-            # without the `-u` flag.
-            call("kill $(ps aux | grep '[p]ython.*main.py' | awk '{print $2}')", shell=True)
+            # Kill the SeedSigner process; Running the process again.
+            # `.*` is a wildcard to detect either `python`` or `python3`.
+            call("kill $(pidof python*) & python /opt/src/main.py", shell=True)
```

Comment:
Temporarily, I've put a soft link at /opt/src pointing to where my seedsigner is installed.  

I notice that this "kill myself and then restart" is a different approach than what I've installed, which uses systemd to always restart seedsigner.   I'm thinking I prefer this self contained restart to relying on systemd... but how exactly should the installation be modified (since I'm trying to track latest dev changes)?